### PR TITLE
add resetPowerMaxMonth, getMagneticFieldThreshold, getMeterInfo commands

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/mtx/commands/uplink/getMagneticFieldThreshold.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ *
+ * @param parameters - command parameters
+ * @returns full message (header with body)
+ */
+export const toBytes = ( parameters: IMagneticFieldThresholdParameters ): types.TBytes => {
+    const buffer: ICommandBinaryBuffer = new CommandBinaryBuffer(maxSize);
+
+    buffer.setUint16(parameters.induction);
+    buffer.setUint16(parameters.threshold);
+    buffer.setUint16(Math.round(parameters.inductionCoefficient * 100));
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/getMagneticFieldThreshold.ts:117]:
```suggestion
    buffer.setUint16(parameters.inductionCoefficient * 100);
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: tests/mtx/message.test.ts [hunk 2]
```diff
@@ -412,6 +412,37 @@ const downlinkMessages: TMessageList = [
         source: 0xffff,
         destination: 0xaaaa
     },
+    {
+        name: 'getMagneticFieldThreshold',
```
Comment (DarkPark) [inline tests/mtx/message.test.ts:416]:
```suggestion
        // does not match empro
        name: 'getMagneticFieldThreshold',
```
?

File: src/mtx/commands/uplink/getMeterInfo.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetMeterInfo.md#response)
+ */
+
+import * as command from '../../utils/command.js';
+import * as types from '../../types.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface IGetMeterInfo {
+    /** Integration period for load profiles */
```
Comment (DarkPark) [inline src/mtx/commands/uplink/getMeterInfo.ts:32]:
```suggestion
    /** Integration period for load profiles (also present in {@link IOperatorParameters}) */
```

and don't forget to import IOperatorParameters

File: src/mtx/commands/uplink/getMagneticFieldThreshold.ts (hunk 1 above)
Comment (aleks-and) [reply src/mtx/commands/uplink/getMagneticFieldThreshold.ts]: fixed

File: src/mtx/commands/uplink/getMeterInfo.ts (hunk 3 above)
Comment (aleks-and) [reply src/mtx/commands/uplink/getMeterInfo.ts]: done

**Review by DarkPark — APPROVED** [review]

**Pull request merged**